functional-tests/test_tracing.py

```diff
@@ -0,0 +1,115 @@
+# SPDX-License-Identifier: GPL-2.0+
+
+from unittest.mock import MagicMock, patch
+
+import stomp
+
+from greenwave.listeners.resultsdb import ResultsDBListener
+from greenwave.subjects.subject import SubjectType
+
+JSON_MESSAGE = {
+    "data": {
+        "brew_task_id": [
+            "57212843"
+        ],
+        "ci_docs": [
+            "https://docs.engineering.redhat.com/display/CVP/" +
```

Comment:
Please replace the internal URLs/mail-addresses/data with some dummy ones (e.g. `https://docs.example.com/ci`).

Reply:
Changed all the values that could lead to somewhere or would give away some information.